File: leo/10808.js
/**
 *  입력 : 최대 100개
 *  제한사항 : 1초
 *  시간계산 필요 : X
 */

const fs = require('fs');
const input = fs.readFileSync(process.platform === "linux" ? "/dev/stdin" : "./input.txt").toString().trim()

// 26길이 딕셔너리 초기화
const result = Array.from({ length: 26 }).fill(0)

// 개수 카운트
for (el of input) {
    const targetIndex = el.charCodeAt() - 'a'.charCodeAt()
    result[targetIndex] += 1;
}

// 공백 포함 출력
console.log(result.join(' '))

File: leo/week13/10808.py
# 문자열 각 알파벳의 개수를 센다.
# 리스트 컴프리헨션으로 리스트 초기화.
# map() 함수로 리스트의 요소를 문자열로 바꾼다.
array = input()
result = [0 for _ in range(26)]

for i in array:
    result[ord(i) - ord('a')] += 1

print(" ".join(map(str, result)))